Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM registry.access.redhat.com/ubi8/ubi
+
+ENV node_version=v12.18.2
+
+RUN yum install xz python3 make -y
+
+RUN curl -L https://nodejs.org/dist/${node_version}/node-${node_version}-linux-x64.tar.xz  -o node-${node_version}-linux-x64.tar.xz && \
+    tar -C /root/ -xf node-${node_version}-linux-x64.tar.xz
+
+ENV PATH=/root/node-${node_version}-linux-x64/bin:$PATH
+
+COPY . /kie-github-ci-bot/
```

Comment:
I would do
```
RUN npm install -g yarn
COPY . /tmp/src/
RUN yarn global add "file:/tmp/src" && rm -rf /tmp/src

ENTRYPOINT [ "kie-github-ci-bot" ]
```

adding 
```
  "bin": {
    "kie-github-ci-bot": "./bin/main.js"
  }
```

Reply:
Didn't we remove yarn dependency from the project?, also how does `ENTRYPOINT [ "kie-github-ci-bot" ]` going to work?

Reply:
I mean how is it going to start our bot server to start listening?

Reply:
`bin/main`should be adapted, or create another one. So please forget my comments, we will do it in another PR